app/retail/templates/shared/head.html

```diff
@@ -38,10 +39,8 @@
 <link rel="stylesheet" href="{% static "v2/css/gitcoin.css" %}" />
 <link rel="stylesheet" href="{% static "v2/css/top-nav.css" %}" />
 <link rel="stylesheet" href="{% static "v2/css/base.css" %}" />
-{% if not slim %}
-  <link rel="stylesheet" href="{% static "v2/css/jquery-ui.css" %}" />
-  <link rel="stylesheet" href="{% static "v2/css/jquery.modal.min.css" %}" />
-{% endif %}
+<link rel="stylesheet" href="{% static "v2/css/jquery-ui.css" %}" />
+<link rel="stylesheet" href="{% static "v2/css/jquery.modal.min.css" %}" />
```

Comment:
adding modal and jquery-ui to all the pages will break a lot of pages.

Reply:
how does it break? wouldnt pages without a modal / jquery-ui just not activate the features in this css files?

Reply:
jquery-ui modals styles overwrite bootstrap ones because they use general class like `.modal` without descendent of special class. There left just a few places  where we using jquery-ui the ideas is totally remove it. I think if you open any modal like submit work you will get a messed modal.

Reply:
got it - is there a task to remove the jquery-ui dependancies?  id add some bounty funds to it

Reply:
i wonder if there are other features of jquery ui we are using...

Reply:
I think just modals and tooltips

Reply:
3601aa02186dea50ab5276d6ffff2ee78e4c66a8